### PR TITLE
Allow choosing a layout of diagrams generated by Graphviz

### PR DESCRIPTION
**Pull request by knu**
Hopefully this could mitigate the issue #1091.

### CODE REVIEW

File: app/helpers/dot_helper.rb [hunk 1]
```diff
@@ -158,7 +158,10 @@ def agent_edge(agent, receiver)
       end
 
       block('digraph', 'Agent Event Flow') {
-        # statement 'graph', rankdir: 'LR'
+        layout ||= ENV['DIAGRAM_DEFAULT_LAYOUT'].presence
```
Comment (cantino) [inline app/helpers/dot_helper.rb]: Should this have a default for if `ENV['DIAGRAM_DEFAULT_LAYOUT']` isn't set?

Comment (knu) [inline app/helpers/dot_helper.rb]: If it isn't set, the graph statement is omitted and Graphviz (or Google Chart API) uses its default layout.  You may of course explicitly add ` || 'dot'`, though.

Comment (cantino) [inline app/helpers/dot_helper.rb]: Ah, I see.  I guess it's fine without.

**Comment by cantino** [comment]
Nice!

**Pull request merged**